tests/ember_debug/view-debug-test.js

```diff
@@ -84,79 +81,6 @@ function isVisible(element) {
   return width > 0 && height > 0;
 }
 
-function setupApp() {
-  this.owner.register('route:application', EmberRoute.extend({
-    model() {
-      return EmberObject.create({
-        toString() {
-          return 'Application model';
-        }
-      });
-    }
-  }));
-
-  this.owner.register('route:simple', EmberRoute.extend({
-    model() {
-      return EmberObject.create({
-        toString() {
-          return 'Simple Model';
-        }
-      });
-    }
-  }));
-
-  this.owner.register('route:comments.index', EmberRoute.extend({
-    model() {
-      return A(['first comment', 'second comment', 'third comment']);
-    }
-  }));
-
-  this.owner.register('route:posts', EmberRoute.extend({
-    model() {
-      return 'String as model';
-    }
-  }));
-
-  this.owner.register('controller:application', Controller.extend({
-    toString() {
-      return 'App.ApplicationController';
-    }
-  }));
-
-  this.owner.register('controller:simple', Controller.extend({
-    toString() {
-      return 'App.SimpleController';
-    }
-  }));
-
-  this.owner.register('component:test-foo', EmberComponent.extend({
-    classNames: ['simple-component'],
-    toString() {
-      return 'App.TestFooComponent';
-    }
-  }));
-
-  this.owner.register('component:test-bar', EmberComponent.extend({
-    tagName: '',
-    toString() {
-      return 'App.TestBarComponent';
-    }
-  }));
-
-  /*
-    Setting line-height to normal because normalize.css sets the
-    html line-height to 1.15. This seems to cause a measurement
-    error with getBoundingClientRect
-  */
-  setTemplate.call(this, 'application', hbs`<div class="application" style="line-height: normal;">{{outlet}}</div>`);
```

Comment:
Do we no longer need these templates for testing?

Reply:
Never mind, just merged the other PR that replaced these 😄

Reply:
This PR just moves them around, but the other one removed the whole concept.